chapters/project_guide.rst

```diff
@@ -1,153 +1,161 @@
+The next thing you should take care of is the list of requirements. Create a file **requirements.txt**. In this file you will specify the required dependencies your Jina project needs. State one module per line. You can then install all the packages using `pip -r`:
 
 .. highlight:: bash
 .. code-block:: bash
 
     pip install -r requirements.txt
 
 Prepare and save data
-----------------------
-
-This can be optional depending on if you need extra data on your project or not. If you need to download data the best practice is to use a script. This script should live directly under the main folder.
+---------------------
 
 ``/get_data.sh``
 
-Now you need someplace where to store the data you just downloaded. For this, you'll create a folder named **data** and inside this folder will live whatever data you downloaded with the previous script. In this example, we have a **data.txt** text file. But this can be whatever you need.
+This step may be necessary and depends on whether you need additional data in your project or not. If you need to download data, the best practice is to use a suitable script. Store this script directly under the main folder.
+
+Now you need to define a place where to store the data you just downloaded. Create a folder named **data** first. Inside this folder will live whatever data you downloaded with the previously created script. 
 
 ``/your-new-project/data/data.txt``
 
+In this example the file is simply named **data.txt**, and it is a text file. You are free to do it and it can be whatever you need for your project.
+
 Flows
----------
+-----
 
 ``/flows``
 
-You will most likely need at least one :term:`Flow`, and it's good practice to have all your Flows in one dedicated folder. To try to be the most explicit as possible, we call this folder also **flows**. In this example, we have two flows, one for index **index.yml** and one for search **query.yml**, but you can have more or less.
+You will most likely need at least one :term:`Flow`, and it is good practice to have all your Flows in one dedicated folder. To be as explicit as possible, we call the folder this **flows**. 
+
+In this example we have two flows -- the first one is for the index that is stored in **index.yml**, and a second one for search that is stored in **query.yml**. In your project you can define as many flows as you actually need.
 
 Pods
----------
+----
 
 ``/pods``
 
 Our Flows will need some :term:`Pods<Pod>`, so we also create a dedicated `pods` folder for them.
 
 App.py
----------
+------
 
 ``/app.py``
 
-And of course, we need our main app, we have this file living directly under the main directory.
+Finally, the main app needs a dedicated place.,This file will be living directly under the main directory.
```

Comment:
```suggestion
Finally, the main app needs a dedicated place. This file will be living directly under the main directory.
```

Reply:
Oh yes, the comma needs to be removed. I will fix that quickly.